ppb/__init__.py

```diff
@@ -1,4 +1,5 @@
 import logging
+import events
 from typing import Callable
 
```

Comment:
```suggestion

from ppb import events
```

Reply:
Should I change the `import events` to `from ppb import events` ?

Reply:
I tried this and it passed the 9 tests!